app/controllers/meals_controller.rb

```diff
@@ -7,8 +7,9 @@ class MealsController < ApplicationController
   def index
     @meals = current_user.meals.includes(:recipe)
     @meals_json = @meals
+      .sort_by(&:time_position)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.